### DISCUSSION TITLE
visualisation of dynamic files

### DISCUSSION THREAD

**Discussion by Q-R-B** [hajkmap/Hajk#1260] [Ideas]
Hi, first thanks for this great tool.

I am wondering  if it would be possible to visualise animated maps.
For flood results that evolve with time, visualising an animation can bring much more understanding of the situation compared to only seing maximal results.

![Torslanda](https://user-images.githubusercontent.com/103583383/210370641-ae0cf154-ad91-4cda-8428-d1eb3e7b50ed.gif)

thanks
Q

**Comment by jacobwod** [comment]
Hi and thanks! 😄 

Did you have a look at the TimeSlider plugin? Our wiki is unfortunately still Swedish-only (but it looks as you're based in Sweden too so that shouldn't be a problem, otherwise please try some translation tool). You can find the documentation here: https://github.com/hajkmap/Hajk/wiki/Tidslinje.

**Reply by Q-R-B** [reply]
Fantastic, i will have a look at that.

thanks for the quick reply :)
